ko/sql/tuning.rst

```diff
@@ -3830,6 +3830,100 @@ N:1 관계의 **LEFT OUTER JOIN**\에서 조인 조건 외에 오른쪽 테이
                 8           60          600  'Left- 706'                     6
                 9           60          600  'Left- 806'                     6
                10           60          600  'Left- 906'                     6
+.. _pred_push
+
+조건절 푸시 최적화
+-----------------------
+조건절 푸시(Predicate Push)는 뷰를 조회하기 전에 먼저 조건절을 적용한다. 
+
+이를 통해 조건에 만족하는 더 적은 양의 데이터만 조회되도록 하여 전체 처리량을 줄일 수 있다.
+
+
+예를 들어 **GROUP BY**\절을 포함한 아래 뷰를 처리할 때, 쿼리 블록 밖에 있는 조건절을 쿼리 블록 안쪽에 밀어 넣을 수 있다면
+**GROUP BY**\  해야할 데이터양을 줄일 수 있다. 
+
+
+.. code-block:: sql
+
+
+    SELECT b.deptno, b.dname, a.avg_sal
+
+    FROM (SELECT deptno, avg(sal) avg_sal FROM emp GROUP BY deptno) a
+
+        , dept b
+
+    WHERE a.deptno = b.deptno
+
+    AND a.deptno = 30;
```

Comment:
I think the empty line is unnecessary.
Was it intended?

Reply:
Nope, I removed empty line